README.md

```diff
@@ -2,9 +2,11 @@
 
 **Deis Workflow** is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any [Kubernetes][k8s-home] cluster, making it easy to deploy and manage applications.
 
+Deis Workflow is the second major release (v2) of the Deis PaaS. If you are looking for the CoreOS-based PaaS visit [https://github.com/deis/deis](https://github.com/deis/deis).
```

Comment:
quick shortcut: you can surround a link with angle brackets to hyperlink it. For example: <http://zombo.com>